internal/options/options.go

```diff
@@ -1,337 +1,180 @@
 package options
```

Comment:
This file was essentially rewritten.  I would recommend reviewing it as if it were a completely new file.